ludwig/models/trainer.py

```diff
@@ -241,7 +241,13 @@ def __init__(
         self.epochs = epochs
         self.regularization_lambda = regularization_lambda
         self.regularization_type = regularization_type
-        self.learning_rate = learning_rate
+        self.learning_rate = learning_rate  # The learning_rate specified in configuration, may be 'auto'.
+        try:
+            target_learning_rate = float(learning_rate)
+        except ValueError:
+            # TODO (ASN): Circle back on how we want to set default placeholder value
+            target_learning_rate = 0.001  # Default initial learning rate for autoML.
```

Comment:
Is this a general default or is this the default for automl's "auto"?

Reply:
Either case we should probably place it somewhere in defaults.py and import it

Reply:
This constant was in `tune_learning_rate()` before, I just moved it up here.  It happens to be the same value as `default_training_params.learning_rate` in `defaults.py`, but has different semantics.